declarations/plugins/DotenvPlugin.d.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file was automatically generated.
+ * DO NOT MODIFY BY HAND.
+ * Run `yarn fix:special` to update
+ */
+
+export interface DotenvPluginOptions {
+	/**
+	 * Whether to allow empty strings in safe mode. If false, will throw an error if any env variables are empty (but only if safe mode is enabled).
+	 */
+	allowEmptyValues?: boolean;
+	/**
+	 * Adds support for dotenv-defaults. If set to true, uses ./.env.defaults. If a string, uses that location for a defaults file.
+	 */
+	defaults?: boolean | string;
+	/**
+	 * Allows your variables to be "expanded" for reusability within your .env file.
+	 */
+	expand?: boolean;
```

Comment:
I think we don't need this option, let's always expand variables, almost all developers use it...